src/test/resources/solr/dssolr/conf/schema.xml

```diff
@@ -490,6 +490,13 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?example     Comedy series?>
   </field>
 
+  <!-- Internal fields for radio/tv -->
```

Comment:
Internal to whom? Maybe mention that the fields are _primarily_ used internally by the Royal Danish Library, but are not classified and might be of use to others.